common/styles/appfooter.css

```diff
@@ -0,0 +1,12 @@
+footer .container1 {
```

Comment:
Styling is being kept separate to make the footer totally independent module. It does not depend upon any existing files.